x-pack/packages/kbn-securitysolution-data-table/common/types/data_table/index.ts

```diff
@@ -6,7 +6,6 @@
  */
 
 import * as runtimeTypes from 'io-ts';
-import type { VIEW_SELECTION } from '../../constants';
 
 export enum Direction {
```

Comment:
This `Direction` is exported but it is never used outside, and we have also another `Direction` imported here:

https://github.com/elastic/kibana/blob/76eddfa817576f0ed121f674d0cb3c5c0ca6186f/x-pack/packages/kbn-securitysolution-data-table/common/search_strategy/common/index.ts#L16

Could we remove this duplicated definition and use the imported one?

Reply:
done